### ISSUE TITLE
Proposal: Add method for mapping

### ISSUE THREAD

**Issue by pyAndr3w** [everx-labs/TVM-Solidity-Compiler#143]
At the moment, to remove an element from mapping, we can only use
```solidity
delete mapping[key]
```
---
However, in TVM there is an opcode for simultaneous deletion and retrieval of an element - `DICTDELGET`.

It would be very convenient to use this method to save on gas.

---

expected syntax:
```solidity
<map>.getDel(KeyType key) returns (optional(ValueType));
```

**Comment by cryshado** [comment]
@pyAndr3w Hi, thank you for the issue! We plan to implement this in the next version

**Comment by cryshado** [comment]
Implemented in 0.71.0 (2023-07-20) #141
